### PR TITLE
Add release note for 1.1.0

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

**Comment by minwoox** [comment]
This does not include #3057 yet. 😉

**Comment by trustin** [comment]
How about including #3024?

File: site/src/pages/release-notes/1.1.0.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  ```java
+  ServerBuilder sb = ...
+  sb.decorator(LoggingService.builder()
+                             .logger("com.foo")
+                             .build());
+  ```
+
+## 📈 Improvements
+
+- Span name is the service name if you use the default `HttpRequestParser` and `HttpResponseParser` with
```
Comment (trustin) [inline site/src/pages/release-notes/1.1.0.mdx:24]: How about:
```
`RequestLog#serviceName()` is now used as a span name if you use ... for <why it's better>.
```

File: site/src/pages/release-notes/1.1.0.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  `BraveClient` and `BraveService`. #3023
+- You can now easily find out whether SPI service interfaces are applied from the logs. #3039
+- You can now get the log of the exception which is raised while making request and response logs. #3043
```
Comment (trustin) [inline site/src/pages/release-notes/1.1.0.mdx:27]: while converting a `RequestLog` into a `String`?

File: site/src/pages/release-notes/1.1.0.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- The stack-trace of <type://HttpStatusException> and <type://HttpResponseException> are not logged in
```
Comment (trustin) [inline site/src/pages/release-notes/1.1.0.mdx:28]: - stack-trace -> stack trace
- not logged anymore by 
- Explain why we don't do that anymore. Depending on the explanation, could it be a bug fix?

File: site/src/pages/release-notes/1.1.0.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  <type://LoggingService>. #3053, #3054
+
+## 🛠️ Bug fixes
+
+- <type://TimeoutScheduler> is initialized exactly once. #3066
```
Comment (trustin) [inline site/src/pages/release-notes/1.1.0.mdx:33]: is now initialized

File: site/src/pages/release-notes/1.1.0.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+- Reactor subscriber context is now properly propagated. #3058
+- Armeria Spring boot autoconfigure collects metrics correctly if <type://ArmeriaSettings#enableMetrics>
```
Comment (trustin) [inline site/src/pages/release-notes/1.1.0.mdx:35]: boot autoconfigure -> Boot integration ?

File: site/src/pages/release-notes/1.1.0.mdx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+is `true`. #3048
+- You now see the proper exception from WebFlux `WebClient` when <type://CircuitBreakerClient> is applied. #3064
+- Armeria client rejects `PUSH_PROMISE` frame correctly because it doesn't support it yet. #3045
```
Comment (trustin) [inline site/src/pages/release-notes/1.1.0.mdx:38]: - rejects -> now rejects
- a PUSH_PROMISE frame (missing a)

File: site/src/pages/release-notes/1.1.0.mdx [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+- Armeria gRPC server does not raise a `NullPointerException` anymore when a gRPC server interceptor closes
+  the `ServerCall`.
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.1.0.mdx:40]: Missing issue link #3032

File: site/src/pages/release-notes/1.1.0.mdx [hunk 8]
```diff
@@ -0,0 +1,86 @@
+---
+date: 2020-09-18
+---
+
+## 🌟 New features
+
+- You can now inject your own <type://MeterIdPrefixFunction> when using Armeria Spring boot autoconfigure. #3034
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.1.0.mdx:7]: Spring boot -> Spring Boot

File: site/src/pages/release-notes/1.1.0.mdx (hunk 6 above)
Comment (ikhoon) [reply site/src/pages/release-notes/1.1.0.mdx:38]: A `PUSH_PROMISE` frame is not rejected. `SETTINGS_ENABLE_PUSH` of the initial settings is set to 0. 

Armeria client tells the server that it doesn't expect `PUSH_PROMISE`. ?

Comment (minwoox) [reply site/src/pages/release-notes/1.1.0.mdx:38]: > A PUSH_PROMISE frame is not rejected.

If a non Armeria server ignores `SETTINGS_ENABLE_PUSH` and sends `PUSH_PROMISE`, we don't reject the frame? then I think we should fix the code. 😄

**Comment by minwoox** [comment]
@trustin and @ikhoon All fixed. 😉

**Review by ikhoon — APPROVED** [review]
Thanks! @minwoox

**Comment by minwoox** [comment]
Thanks for reviewing. 😉

**Pull request merged**